wiki/Skinning/pt-br.md

```diff
@@ -0,0 +1,28 @@
+# Skinning
+
+Skinning é um dos principais recursos do osu!. Possibilita os jogadores derivarem os elementos originais de skins para criar a sua própria! Skins podem variar desde temáticas, profissionais, quase impossível de usa-las ou apenas para diversão.
```

Comment:
`derivarem` is just too formal, and this way would be better for understanding.
```suggestion
Skinning é um dos principais recursos do osu!. Possibilita que os jogadores utilizem os elementos originais de skins para criar a sua própria! Skins podem variar desde temáticas, profissionais, quase impossível de usá-las ou apenas para diversão.
```

Reply:
nice thx